instawow/cli.py

```diff
@@ -215,6 +215,33 @@ def install(manager, addons, strategy, replace) -> None:
     Report(list(results)).generate_and_exit()
 
 
+@main.command()
+@click.argument('search_pattern', nargs=-1, required=True)
+@click.pass_context
+def search(ctx, search_pattern) -> None:
+    "Search add-ons."
+    manager = ctx.obj
+    search_pattern = ' '.join(search_pattern)
+    addons = manager.search(search_pattern)
+    for i, addon in reversed(list(enumerate(addons, 1))):
+        print('{i}. {name} (by {author})\n     {desc}'.format(i=i, **addon))
```

Comment:
Though this only works with Curse (for the time being) users won't know that so it'd make sense to show the source in every search result.